src/zenml/cli/stack.py

```diff
@@ -261,7 +278,7 @@ def register_stack(
 )
 @click.option(
     "-d",
-    "--model-deployer",
+    "--model_deployer",
```

Comment:
This change definitely implies a sweep through the docs to make sure we didnt break anything

Reply:
You're right, this might break a lot of stuff. I suggest we undo this change, IMO we should do it in a separate PR. What do you think?

Reply:
Ok makes sense!

Reply:
Could you make a PR for it please? :-)

Reply:
Lets leave it in!